opm/parser/eclipse/EclipseState/Schedule/SegmentSet.hpp

```diff
@@ -0,0 +1,106 @@
+/*
+  Copyright 2015 SINTEF ICT, Applied Mathematics.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef SEGMENTSET_HPP_HEADER_INCLUDED
+#define SEGMENTSET_HPP_HEADER_INCLUDED
+
+#include <vector>
+#include <map>
+
+#include <opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/Segment.hpp>
+
+namespace Opm {
+
+    class SegmentSet {
+    public:
+        SegmentSet();
+
+        std::string wellName() const;
+        int numberBranch() const;
+        int numberSegment() const;
+        double depthTopSegment() const;
+        double lengthTopSegment() const;
+        double volumeTopSegment() const;
+        double xTop() const;
+        double yTop() const;
+
+        WellSegment::LengthDepthEnum lengthDepthType() const;
+        WellSegment::CompPresureDropEnum compPressureDrop() const;
+        WellSegment::MultiPhaseModelEnum multiPhaseModel() const;
+        std::vector<SegmentPtr>& Segments();
```

Comment:
I forgot the reason, probably to reduce the length of each line of the code. I will fix it.